src/components/LeftSidebar/LeftSidebar.vue

```diff
@@ -111,17 +111,18 @@
 		</div>
 
 		<template #list>
-			<li ref="container" class="left-sidebar__list" @scroll="debounceHandleScroll">
-				<ul class="scroller">
+			<li ref="container" class="left-sidebar__list">
+				<ul class="scroller h-100">
 					<!-- Conversations List -->
 					<template v-if="!isSearching">
```

Comment:
UPD: using `v-show` and keeping scroller in DOM helps with keeping the scrol position

Reply:
Still thinking, that it's an issue: try scroll to the bottom and use search from there